### PR TITLE
Add cronjob trigger to OSP nightly workflow

### PR DESCRIPTION
**Pull request by enarha**
The intention always was to use a scheduler to execute the workflow daily. The manual trigger is kept for both easier testing and emergency updates.

### CODE REVIEW

**Review by gabemontero — APPROVED** [review]
/lgtm

am I correct in thinking that the only proper verification of this @enarha is to look for the github action firing at the appointed interval?

**Comment by enarha** [comment]
> am I correct in thinking that the only proper verification of this @enarha is to look for the github action firing at the appointed interval?

Yes. If that helps, we have similar setup here https://github.com/openshift-pipelines/pipeline-service/blob/main/.github/workflows/periodic-scanner-quay.yaml#L3-L6 and I tested it myself.

**Comment by gabemontero** [comment]
> > am I correct in thinking that the only proper verification of this @enarha is to look for the github action firing at the appointed interval?
> 
> Yes. If that helps, we have similar setup here https://github.com/openshift-pipelines/pipeline-service/blob/main/.github/workflows/periodic-scanner-quay.yaml#L3-L6 and I tested it myself.

cool ... I'll defer to you or @Roming22 on when to hit the merge button

**Pull request merged**
